waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -45,3 +45,22 @@ export const invalidateQueriesUsing = (resources) => {
     queryClient.invalidateQueries(queryCacheKey)
   }
 }
+
+export function removeQueries() {
+  // TODO(filip): We are currently removing all the queries, but we should
+  // remove only non-public, user-dependent queries - public queries are
+  // expected not to change in respect to the currently logged in user.
+  queryClient.removeQueries()
+}
+
+// TODO(filip): We are currently invalidating and removing  all the queries, but
+// we should remove only the non-public, user-dependent ones.
```

Comment:
Why? This comments also feels out of context. Especially since function is called `invalidateAndClearQueries`.

Reply:
Let's resolve this one and continue the discussion in the related comment: https://github.com/wasp-lang/wasp/pull/595/files/bd946bcbd818bf3e96ad28e02d06968aff4c6d4e#r878000018